pkg/pool-weighted/contracts/lbp/LBPool.sol

```diff
@@ -156,6 +164,11 @@ contract LBPool is ILBPool, WeightedPool, Ownable2Step, BaseHooks {
             lbpParams.reserveTokenEndWeight
         );
 
+        _lockDurationAfterMigration = lockDurationAfterMigration;
+        _bptPercentageToMigrate = bptPercentageToMigrate;
+        _migrationWeightProjectToken = migrationWeightProjectToken;
+        _migrationWeightReserveToken = migrationWeightReserveToken;
```

Comment:
Let's also check these parameters here.

Migration will fail if `_migrationWeightProjectToken + _migrationWeightReserveToken != FixedPoint.ONE`, and that would lock the liquidity if I'm not mistaken since we won't allow regular exits (well, we would still have recovery mode but that's not ideal).

So it's better to fail early and disallow creating the LBP if the migration parameters are incorrect.

Also the lock duration should have a maximum duration (1 year? 4 years? But not more than that).
BPT percentage to migrate should be < FP(1) too.